pipeline_steps/common.groovy

```diff
@@ -409,10 +410,10 @@ def archive_artifacts(Map args = [:]){
       // the build result to be set to failure.
       // try-catch placed here because this function is used in multiple places.
       String jiraProjectKey = args.get("jiraProjectKey", "RE")
-      if (jiraProjectKey){
+      print "Error while archiving artifacts, swallowing this exception to prevent "\
```

Comment:
It might make sense to turn the if block on line 425 to and if/else block and add this to the else.